libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1460,11 +1460,11 @@ GCS_MAVLINK::update_receive(uint32_t max_time_us)
 
     status.packet_rx_drop_count = 0;
 
+    const uint32_t protocol_timeout = 4000;
```

Comment:
Compiler output is identical before/after this change on Durandal.

We're moving protocol_timeout further from where it is used - it's not a stellar name for a variable in the first place, and I kind of think this makes the code marginally less readable, sorry.